src/pages/signup/Signup.tsx

```
@@ -0,0 +1,41 @@
+import styled from 'styled-components'
+import { AuthLayout } from '../../components/AuthLayout'
+import First from './First'
+import { useState } from 'react'
+import Second from './Second'
+import Third from './Third'
+
+function Signup() {
+  const [step, setStep] = useState<number>(2)
+  const signupPage = [<First />, <Second />, <Third />]
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)